Backend/app/provider-platform/dynamic-offer-driver-app/Main/src-read-only/Domain/Types/CoinConfigTranslation.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module Domain.Types.CoinConfigTranslation where
```

Comment:
Why have we made a new table, rather than using the translations table ? @spravinkumar9952